app/models/supabase_client.py

```
@@ -354,7 +357,7 @@ def get_data(
         try:
             response = query.execute()
         except Exception as e:
-            print("SupabaseClient get_data error:", e, flush=True)
+            logger.error(f"[get_data] SupabaseClient get_data error: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)